docs/pages/machine-id/guides/ansible.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Ansible
```

Comment:
I think description is short enough where we could use it as the title as well for some more SEO.